packages/modules/create-imodel/src/components/base-imodel/BaseIModel.tsx

```diff
@@ -273,54 +258,38 @@ export function BaseIModelPage(props: BaseIModelProps) {
-              <LabeledInput
-                label={updatedStrings.nameString}
-                name="name"
-                setFocus
-                required
-                value={imodel.name}
-                onChange={onPropChange}
-                message={
-                  isPropertyInvalid(imodel.name)
-                    ? updatedStrings.nameTooLong
-                    : undefined
-                }
-                status={isPropertyInvalid(imodel.name) ? "negative" : undefined}
-                autoComplete="off"
-              />
-              <LabeledTextarea
-                label={updatedStrings.descriptionString ?? "Description"}
-                name="description"
-                value={imodel.description}
-                onChange={onPropChange}
-                rows={4}
-                message={
-                  isPropertyInvalid(imodel.description)
-                    ? updatedStrings.descriptionTooLong
-                    : undefined
-                }
-                status={
-                  isPropertyInvalid(imodel.description) ? "negative" : undefined
-                }
-                autoComplete="off"
-              />
-              {!extentComponent && (
+          {props?.children ?? (
+            <div className="iac-imodel-properties-container">
+              <div className="iac-inputs-container">
                 <>
-                  {PointInput(updatedStrings.southWestCoordinate, "southWest")}
-                  {PointInput(updatedStrings.northEastCoordinate, "northEast")}
+                  <IModelName
+                    nameString={updatedStrings?.nameString}
+                    nameTooLong={updatedStrings?.nameTooLong}
+                  />
+                  <IModelDescription
+                    descriptionString={updatedStrings?.descriptionString}
+                    descriptionTooLong={updatedStrings?.descriptionTooLong}
+                  />
+                  {!extentComponent && (
+                    <>
+                      {PointInput(
+                        updatedStrings.southWestCoordinate,
+                        "southWest"
+                      )}
+                      {PointInput(
+                        updatedStrings.northEastCoordinate,
+                        "northEast"
+                      )}
+                    </>
+                  )}
+                  <UploadImage />
                 </>
+              </div>
+              {extentComponent && (
+                <div className="iac-extent-container">{extentComponent}</div>
               )}
-              <UploadImage
-                onChange={onImageChange}
-                src={imodel.thumbnail?.src}
-              />
             </div>
-            {extentComponent && (
-              <div className="iac-extent-container">{extentComponent}</div>
-            )}
-          </div>
+          )}
         </div>
         <div className="iac-button-bar">
```

Comment:
Maybe it is worth exposing action bar in case someone wants a bit different design.

Reply:
yes, can we handle this in new PBI then?

Reply:
But then it will be a breaking change because if you are passing `children` then button bear will disappear.

Reply:
I would even say to have fully custom content if `children` is passed, including custom title too.

Reply:
If I guess right what you are trying to say here,
we have buttons not included in condition whether there are children passed or not
we still using the button bar from package itself

Reply:
Yes, but if we later move them to separate component, when using `children`, button bar would not be shown.